src/balances/utils.int.spec.ts

```diff
@@ -78,5 +79,13 @@ describe('balances utils', () => {
       const tokens: Token[] = []
       await loadAndCompareTokenAmounts(walletAddress, tokens)
     })
+
+    it('should handle large lists', async () => {
```

Comment:
maybe specify what a large list is? 
in this case it's more than 100 entries. Everything below that would work just fine.

'should handle lists with more then 100 entries'